SEED-platform/seed
# Feat/populate sample data

#### Any background context you want to provide?
This is specifically for new users to allow them the opportunity to learn and use the SEED Platform

#### What's this PR do?
This PR specifically updates the Download Sample Data files to include Interval data.  This PR also creates an API that will be hit for new users to have sample data auto-populated in SEED if they do not have an organization.  The data includes:
- 1 record for Geo-coding
- 2 merged records
- 1 record with building footprints
- 1 record with an associated tax-lot
- 4 records with labels
- 2 records with interval data
- 1 record with green button imported data

#### How should this be manually tested?
Check the API creates the records above by entering your organization.  Soon a button will be created on the landing page to do this automatically for new users once they click it.

#### What are the relevant tickets?
#2595
